homework/homework_part_2/javascript/app.js

```diff
@@ -0,0 +1,103 @@
+//1. With interpolation you apply your variable inside 
+// of a string and concatenation you write everything out 
+// as a string 
+// INTERPOLATION: const red = 2
+//const blue = 5
+//console.log(`Hello World ${red + blue}`)
+//CONCATENATION: console.log("Hello World" +  " AND GOOD DAY");
+
```

Comment:
more precisely the difference is that interpolation means that you're creating a string from variables that contain strings (not just string literals)